src/components/header-footer.tsx

```diff
@@ -11,7 +11,9 @@ export function HeaderFooter({ children }: HeaderFooterProps) {
   return (
     <>
       <Header />
-      <div className="min-h-[68vh] max-w-full overflow-x-hidden">
+      <div
+        className="min-h-[68vh] max-w-full overflow-x-hidden lg:overflow-visible" /* needed for sticky menu*/
```

Comment:
@elbotho THIS could be the final approach :)

Reply:
only thing that breaks is the sticky header for e.g. https://de.serlo.org/entity/repository/compare/224015/224311 - but that could be fine.

Reply:
since it's working on larger screens I can live with that.